Sources/Task.swift

```diff
@@ -76,15 +75,14 @@ extension Task: Hashable {
 			&& lhs.environment == rhs.environment
 	}
 
-	public var hashValue: Int {
-		var result = launchPath.hashValue ^ (workingDirectoryPath?.hashValue ?? 0)
-		for argument in arguments {
-			result ^= argument.hashValue
+	public func hash(into hasher: inout Hasher) {
+		hasher.combine(launchPath)
+		hasher.combine(workingDirectoryPath)
+		arguments.forEach { hasher.combine($0) }
+		(environment ?? [:]).forEach { key, value in
+			hasher.combine(key)
+			hasher.combine(value)
```

Comment:
I am not sure if the following can be used instead?

    hasher.combine(launchPath)
    hasher.combine(workingDirectoryPath)
    hasher.combine(arguments)
    hasher.combine(environment)